### PR TITLE
Added Styling, and made some small changes to the edit button

### PR DESCRIPTION
**Pull request by zjhellweg**
Added css and a main page.
Added some padding to the add task button, along with a dynamic title.

### CODE REVIEW

**Review by ConnieZi — APPROVED** [review]

File: src/DashboardPage/DashboardPage.js [hunk 1]
```diff
@@ -83,7 +127,25 @@ function DashboardPage() {
 
   return (
     <div>
-      <h1>Dashboard</h1>
+      <div class="topnav">
```
Comment (PhuongDTran) [inline src/DashboardPage/DashboardPage.js:130]: `className` should be used instead of `class`

File: src/DashboardPage/DashboardPage.js [hunk 2]
```diff
@@ -32,6 +34,27 @@ function DashboardPage() {
     }
   }, []);
 
+  function handleSortSelecter(Picked) {
```
Comment (PhuongDTran) [inline src/DashboardPage/DashboardPage.js:37]: use lower case params. same for others

File: src/DashboardPage/DashboardPage.js [hunk 3]
```diff
@@ -32,6 +34,27 @@ function DashboardPage() {
...
+    localStorage.setItem("Picked", Picked);
+    document.location.reload();
+  }
+
+  function StartingAssignment() {
```
Comment (PhuongDTran) [inline src/DashboardPage/DashboardPage.js:42]: function names should start with lower case.

File: src/DashboardPage/DashboardPage.js [hunk 4]
```diff
@@ -32,6 +34,27 @@ function DashboardPage() {
...
+    let checkedBox
+    let item = localStorage.getItem("Picked")
+
+    setTimeout(() => {
```
Comment (PhuongDTran) [inline src/DashboardPage/DashboardPage.js:46]: what is this timeout for?

File: src/DashboardPage/DashboardPage.js [hunk 5]
```diff
@@ -32,6 +34,27 @@ function DashboardPage() {
...
+      if (item) {
+        checkedBox = document.getElementById(item);
+        checkedBox.checked = true;
+      } else {
+        localStorage.setItem("Picked","title");
+        checkedBox = document.getElementById("title");
+        checkedBox.checked = true;
```
Comment (PhuongDTran) [inline src/DashboardPage/DashboardPage.js:53]: checkBox is null if navigate to /dashboard without login. the app crashed

**Review by PhuongDTran — COMMENTED** [review]
Do we have option to sort asc and desc?

**Pull request merged**